src/gt4py/next/program_processors/runners/dace_fieldview/gtir_sdfg.py

```diff
@@ -413,73 +448,35 @@ def _add_sdfg_params(
             # the connectivity tables that are not used. The remaining unused transient arrays
             # are removed by the dace simplify pass.
             self._add_storage(
-                sdfg, symbolic_arguments, dace_utils.connectivity_identifier(offset), gt_type
+                symbolic_arguments, dace_utils.connectivity_identifier(offset), gt_type
             )
 
         # the list of all sdfg arguments (aka non-transient arrays) which include tuple-element fields
         return [arg_name for arg_name, _ in sdfg_args]
 
-    def visit_Program(self, node: gtir.Program) -> dace.SDFG:
-        """Translates `ir.Program` to `dace.SDFG`.
-
-        First, it will allocate field and scalar storage for global data. The storage
-        represents global data, available everywhere in the SDFG, either containing
-        external data (aka non-transient data) or temporary data (aka transient data).
-        The temporary data is global, therefore available everywhere in the SDFG
-        but not outside. Then, all statements are translated, one after the other.
-        """
-        if node.function_definitions:
-            raise NotImplementedError("Functions expected to be inlined as lambda calls.")
+    def visit_Program(self, node: gtir.Program) -> None:
+        """Translates the body of `ir.Program` inside the SDFG that belongs to this builder."""
```

Comment:
OPTIONAL: the description sounds a bit strange, how does:
Translates the program contained in `gtir.Program` into an SDFG.

Reply:
I agree.